.eslintrc

```diff
@@ -15,6 +15,7 @@
     "block-scoped-var": "error",
     "camelcase": "error",
     "comma-dangle": ["error", "always-multiline"],
+    "consistent-this": ["error", "self"],
```

Comment:
Again, not a single one of the lines you've changed as a result of turning this rule on is a true violation of the rule, so what's the point?

More confusingly, this diff is *itself* inconsistent as to whether `vm` is a valid assignment for `this` -- in my opinion it should be, because it makes it possible to read code the same way in both the JavaScript file and in the template (with `controllerAs: "vm"`) -- but in places you've added a pragma, and in others you've got rid of `vm`.